lavalink/__init__.py

```diff
@@ -36,3 +38,39 @@ def enable_debug_logging():
     log.addHandler(handler)
 
     log.setLevel(logging.DEBUG)
+
+
+def _internal_add_event_hook(hook, event: Event = None):
```

Comment:
This function should probably do some kind of type checking. I.e., `if event is not None and not isinstance(event, lavalink.events.Event)` or something.

Reply:
What should it raise?

Reply:
TypeError('Event parameter is not of type `Event`')